app/Http/Controllers/Dashboard/IncidentController.php

```diff
@@ -120,27 +122,33 @@ public function createIncidentAction()
                 null,
                 null
             ));
+            // Add the new incident to incidentHistory
+            $incidentsHistoriesAttributes = array(
```

Comment:
Shouldn't this be called: $incidentHistoryAtrributes or better $incidentAttributes because these are the attributes of one single incident?